README.md

```diff
@@ -30,6 +30,28 @@ Note: The PoC application shown here, is ideal for demonstration purposes only.
 
 ---
 
+## E2E Testing
+
+For e2e testing we are using cypress and **docker image** - `image: "cypress/included:3.2.0"` which comes with preinstalled cypress. Cypress config and tests can be found in `./doocker/` folder. Test flows or Cypress specs are located in `./docker/cypress/integration/` folder. Each spec should represent a flow e.g.
```

Comment:
Do we need additional env configuration for this?

Reply:
No for this repo. At the moment there are no flows, but we will require env for authentication and other stuff which will be added as part of a first test along with the env file